app/jobs/bulkrax/schedule_relationships_job.rb

```diff
@@ -9,7 +9,7 @@ def perform(importer_id:)
       return reschedule(importer_id) unless pending_num.zero?
 
       importer.last_run.parents.each do |parent_id|
-        CreateRelationshipsJob.perform_later(parent_identifier: parent_id, importer_run_id: importer.last_run.id)
+        Bulkrax.relationship_job.perform_later(parent_identifier: parent_id, importer_run_id: importer.last_run.id)
```

Comment:
Not a big deal, but I like the idea of this being called `relationship_job_class`

Reply:
I like that too, I changed the file model variable to use the same name format so it matches.